packages/contracts/test/conditional-transaction.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { Contract, ContractFactory } from "ethers";
+import { AddressZero, HashZero, WeiPerEther, Zero } from "ethers/constants";
+import { JsonRpcSigner, Web3Provider } from "ethers/providers";
+import {
+  defaultAbiCoder,
+  hexlify,
+  randomBytes,
+  solidityKeccak256
+} from "ethers/utils";
+
+import { expect } from "./utils";
```

Comment:
- seems better to me to inline this 
- the definition of `expect` varies through this codebase; is it possible to have it be the same thing everywhere?

Reply:
It's necessarily different in test files that use Jest vs Mocha / Jasmine

Reply:
Yeah, truffle forces usage of Mocha :(